config/routes.rb

```diff
@@ -727,6 +727,7 @@
 
   get "message-bus/poll" => "message_bus#poll"
 
+  resources :drafts
```

Comment:
This should be `resources :drafts, only: [:index]` so that we don't generate the other routes that we don't require.